plugins/module_utils/acme/challenges.py

```diff
@@ -28,6 +27,11 @@
     ModuleFailException,
 )
 
+try:
+    import ipaddress
+except ImportError:
+    pass
```

Comment:
Is this really necessary if it's shielded against being missing on the outer layer?
```suggestion
import ipaddress
```

Reply:
Yes. Also otherwise the sanity import tests will fail, since without this, this file is no longer importable on a Python 2.6 or 2.7 without ipaddress installed.

Reply:
Also, here's a way to do the same that better reflects the semantics of the intent to ignore the importing outcome. It reads as "attempt to make an import but suppress a failure to do so".
```python
import contextlib
```
```suggestion
with contextlib.suppress(ImportError):
    import ipaddress
```
(of course, if the Python runtime has this, I don't remember when this was added)